Example/ViewController.swift

```diff
@@ -108,6 +118,13 @@ class ViewController: UIViewController {
         self.speedLimitView = speedLimitView
     }
     
+    func setupFeedbackButton() {
+        view.addSubview(feedbackButton)
+        feedbackButton.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 12).isActive = true
+        feedbackButton.rightAnchor.constraint(equalTo: view.safeAreaLayoutGuide.rightAnchor, constant: -12).isActive = true
```

Comment:
I guess it doesn't make a lot of difference in this case, because it's the only button on the map and doesn't even contain text. But you are right, for consistency with `MapOrnamentPosition`, I changed the anchor to the trailing.